wdl-lint/src/rules/malformed_lint_directive.rs

```diff
@@ -0,0 +1,111 @@
+//! A lint rule for flagging malformed lint directives.
+use wdl_ast::AstToken;
+use wdl_ast::Comment;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Document;
+use wdl_ast::Span;
+use wdl_ast::SupportedVersion;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+use crate::util::is_inline_comment;
+
+/// The identifier for the Malformed Lint Directive rule.
+const ID: &str = "MalformedLintDirective";
+/// The accepted lint directives.
+const ACCEPTED_LINT_DIRECTIVES: [&str; 1] = ["except"];
+
+/// Creates an "Inline Lint Directive" diagnostic.
+fn inline_lint_directive(
+    span: Span,
+) -> Diagnostic {
+    Diagnostic::warning(
+        "lint directive must be on its own line")
+        .with_rule(ID)
+        .with_label("malformed lint directive", span,
+        )
+        .with_fix("place lint directive on new line")
```

Comment:
re: #193 

I think it's best to leave this without a `with_fix()`. You were probably following the existing examples, which I want to clean up when I get a chance.